pages/data-fetcher.tsx

```diff
@@ -40,14 +45,32 @@ const GetData: NextPage = () => {
   const [addressError, setAddressError] = useState('');
   const [dataKey, setDataKey] = useState(dataKeyList[0].key);
   const [dataKeyError, setDataKeyError] = useState('');
+
   const [data, setData] = useState('');
+  const [decodedData, setDecodedData] = useState('');
+
   const [interfaces, setInterfaces] = useState({
     isErc725X: false,
     isErc725Y: false,
   });
 
+  const [erc725js, setERC725JsInstance] = useState<ERC725>();
```

Comment:
erc is capitalized and then non capitalized maybe we want to have is either capitalised or non capitalized consistently?
```suggestion
  const [erc725js, setErc725jsInstance] = useState<ERC725>();
```